src/helper/network/proto.hpp

```diff
@@ -142,6 +154,47 @@ struct request_init {
     };
 };
 
+struct config_sync {
+    struct request : base_request {
+        static constexpr const char *name = "config_sync";
+        static constexpr request_id id = request_id::config_sync;
+
+        dds::parameter data;
```

Comment:
Data should not be used here and instead the extension should send the value of the APPSEC_ENABLED env